### PR TITLE
Split out remaining components from shell

### PR DESCRIPTION
**Pull request by stefwalter**
Split out the remaining components from the shell package, and cleanup the shell component.

 * system/host
 * dashboard/list

### CODE REVIEW

**Comment by stefwalter** [comment]
There is still some code in shell package that eventually needs to be moved out. @mvollmer's plot patches do some of that.

File: test/check-multi-os [hunk 1]
```diff
@@ -99,7 +104,7 @@ class TestMultiOS(MachineCase):
             stock_m.wait_boot()
             stock_b = self.new_browser(stock_m.address)
 
-            stock_b.login_and_go("dashboard", href="/dashboard/list", host=None)
+            stock_b.login_and_go("/dashboard/list", host=None)
```
Comment (mvollmer) [inline test/check-multi-os]: The stock version of the dashboard still needs the old way, since it hasn't changed.

**Comment by mvollmer** [comment]
check-multi-os fails:

```
ERROR: testBasic (__main__.TestMultiOS)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./check-multi-os", line 124, in testBasic
    test_stock('fedora-22')
  File "./check-multi-os", line 107, in test_stock
    stock_b.login_and_go("/dashboard/list", host=None)
  File "/home/mvo/work/cockpit/test/testlib.py", line 355, in login_and_go
    self.enter_page(page, host=host)
  File "/home/mvo/work/cockpit/test/testlib.py", line 322, in enter_page
    self.wait_present("iframe.container-frame[name='%s'][data-loaded]" % frame)
  File "/home/mvo/work/cockpit/test/testlib.py", line 230, in wait_present
    return self.wait_js_func('ph_is_present', selector)
  File "/home/mvo/work/cockpit/test/testlib.py", line 224, in wait_js_func
    return self.phantom.wait("%s(%s)" % (func, ','.join(map(jsquote, args))), timeout=self.phantom_wait_timeout)
  File "/home/mvo/work/cockpit/test/testlib.py", line 629, in wait
    return self.run({'cmd': 'wait', 'cond': cond, 'timeout': timeout*1000})
  File "/home/mvo/work/cockpit/test/testlib.py", line 592, in run
    raise Error("timeout")
Error: timeout
```

This is becaused the dashboard of the stock version of cockpit still is part of the shell and needs the old code when accessing it.

**Comment by mvollmer** [comment]
Seems to work other than the check-multi-os failure.

**Comment by mvollmer** [comment]
Hubbot/mock/npm is broken in a weird way... I have symlink /var/lib/mock into /home to avoid filling /.  Could that be the reason?

**Comment by stefwalter** [comment]
> Hubbot/mock/npm is broken in a weird way... I have symlink /var/lib/mock into /home to avoid filling /. Could that be the reason?

npm always want to write tmp files to home.

**Comment by stefwalter** [comment]
I think that should fix the check-multi-os failure.

**Comment by stefwalter** [comment]
checkShutdownReboot failure is due to wait_boot() races that @dperpeet is fixing.

**Comment by mvollmer** [comment]
The links next to the graphs on the system/host page don't work:

```
ReferenceError: cockpit is not defined
```

**Comment by stefwalter** [comment]
Fixed server page graphs regression. First commit here is new.

**Pull request closed without merging**